docs/onboarding/8_16/operator/onprem_kube_stack_values.yaml

```diff
@@ -103,13 +112,6 @@ collectors:
                 key: app.kubernetes.io/component
                 from: pod
       receivers:
-        k8sobjects:
```

Comment:
no, otherwise will be used the default value that is defined in helm chart - https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-kube-stack/values.yaml#L552-L555:
```
receivers: [k8sobjects]
processors: [resourcedetection/env, batch]
exporters: [debug]
```
instead we want to have:
```
 receivers:
  - k8sobjects
processors:
  - resourcedetection/eks
  - resourcedetection/gcp
  - resourcedetection/aks
exporters:
  - debug
  - elasticsearch/otel
```
to override it, I've just redefined `service.pipelines.logs`

reason to remove it from `receivers` config - since exactly the same configuration is defined in chart - https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-kube-stack/templates/_config.tpl#L336-L345 and `kubernetesEvents` is enabled in presets https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-kube-stack/values.yaml#L529-L530

Reply:
Good catch, thanks!